src/monitord/monitord.c

```diff
@@ -156,7 +156,7 @@ void Monitord()
     }
 
     // Start com request thread
-    w_create_thread(moncom_main, NULL);
+    w_create_thread(moncom_main, NULL, 8192);
```

Comment:
I think you can create a configuration structure for monitord that contains `thread_stack_size` as you did in all the other daemons.

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/4146/commits/dfea85e7c204bd401063f91edd8c7bcca6d847a7.